app/ai/voice/agents/breeze_buddy/agent/turn_strategies/validators.py

```
@@ -0,0 +1,142 @@
+            from pipecat.audio.turn.krisp_viva_turn import KrispVivaTurn
+
+            return True, None
+
+        else:
+            return False, f"Unknown analyzer type: {analyzer_type}"
+
+    except ImportError as e:
+        error_msg = (
+            f"Analyzer '{analyzer_type}' is not available. "
+            f"Install required dependencies: pip install 'pipecat-ai[{analyzer_type.replace('_', '-')}]'. "
+            f"Error: {str(e)}"
```

Ruff (0.14.14)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)